framework/packages/abstract-std/src/objects/common_namespace.rs

```diff
@@ -1,6 +1,10 @@
 /// namespace key for State Item
-pub const BASE_STATE: &str = "base_state";
+pub const BASE_STATE: &str = "a";
 /// namespace for contract Admin
-pub const ADMIN_NAMESPACE: &str = "admin";
+pub const ADMIN_NAMESPACE: &str = "b";
 /// storage key for cw_ownable::Ownership
-pub const OWNERSHIP_STORAGE_KEY: &str = "ownership";
+pub const OWNERSHIP_STORAGE_KEY: &str = "c";
+/// storage key for Account Id
+pub const ACCOUNT_ID_STORAGE_KEY: &str = "d";
```

Comment:
Let's have it as "id"? It has same key as `ACCOUNT_MODULES` for account right now

Reply:
This was modified in develop/v2 to be included inside account, so this is now a letter in the account namespace